src/content/pages/build/taco.md

```diff
@@ -0,0 +1,213 @@
+tacoInfo:
+  title: "TACo: Privacy without trust"
+  description: "Give your users true sovereignty over their data, today. TACo – the first and only access control layer you aren’t forced to trust."
+  image: /images/taco-encryption.svg
+  buttons:
+    - label: Start Building
+      url: https://docs.threshold.network/app-development/threshold-access-control-tac/integration-guide
+      variant: EXTERNAL_SOLID
+    - label: Trust Overview
+      url: https://docs.threshold.network/app-development/threshold-access-control-tac/trust-assumptions/cbd-mainnet-version
+      variant: EXTERNAL_OUTLINE
+  rowReverse: false
+tacoFeaturesInfo:
+  title: "The Fully Decentralized End-to-End Encryption"
+  description: "Empower users to share data with qualifying recipients, and no-one else Encryption via a decentralized cryptosystem."
+  highlighted: ["Decentralized", "Encryption"]
+tacoFeatures:
+  - title: "Web3 needs TACo"
+    cards:
+      - title: "Decentralized Storage"
+        description: "TACo is the perfect compliment to persistent storage, finally enabling users to grant future access without having to download & re-encrypt locally."
+        icon: /images/storage-icon.svg
+      - title: "Web3 Social Networks"
+        description: "TACo enables many-to-many data sharing, minus a centralized  authority that eventually decides to harvest and monetize interpersonal communication."
+        icon: /images/social-icon.svg
+      - title: "Media content-gating"
+        description: "Whether it's a stream, a movie or a collectible, TACo enforces decryption rights to media, so owning an NFT has meaning beyond a symbolic receipt."
+        icon: /images/media-icon.svg
+  - title: "The Web needs TACo"
+    cards:
+      - title: "Hyper-sensitive data"
+        description: "From reproductive health data to  seed phrase recovery, TACo is leveraged to ensure that high-stakes data sharing doesn’t involve trusting an intermediary."
+        icon: /images/locker-icon.svg
+      - title: "IoT & Connected Vehicles"
+        description: "TACo is the antidote to the interconnected hardware panopticon, where every waking (and sleeping) moment of our lives is surveilled."
+        icon: /images/iot-icon.svg
+      - title: "Activism & Human Rights"
+        description: "TACo protects those most at risk of intimidation, censorship and reprisal based on a data leaks – whether via a hack, subpoena, or a data vending business model."
+        icon: /images/activism-icon.svg
+tacoDiagramsInfo:
+  title: "Integrate a Privacy Layer with Nothing to Hide"
+  description: "TACo isn't just auditable, it's comprehensible, from high-level schematic to client code."
+  image: /images/taco-grid.svg
+  background: /images/glowing-background.png
+tacoDiagrams:
+  - title: "Generating a DKG Cohort"
+    preTitle: "learn"
+    description: "Independent nodes sampled from the Threshold network form a cohort – who collectively generate a DKG public key."
```

Comment:
```suggestion
    description: "Independent nodes are sampled from the Threshold network form a cohort. They collectively generate a DKG public key."
```

Reply:
"form a cohort" should be corrected. How about something like:
```suggestion
    description: "Independent nodes are sampled from the Threshold Network, forming a cohort that collectively generates a DKG public key."
```

Reply:
_Independent nodes sampled from the Threshold network form a cohort_  is actually correct grammatically. But I think your suggestion is more consistent with the other stages so let's go with that. @evandrosaturnino